packages/ilios-common/addon/components/session/objective-list-item-parents.hbs

```diff
@@ -33,7 +33,14 @@
       >
         {{#if @editable}}
           <span data-test-parent>
-            <button type="button" class="link-button" {{on "click" @manage}} data-test-manage>
+            <button
+              type="button"
+              class="link-button"
+              aria-label={{t "general.edit"}}
```

Comment:
Repeat comment, but this probably isn't needed with the title appearing for sure.